core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -3337,6 +3337,15 @@ paths:
       summary: list billing accounts for a user
       description: list billing accounts for a user
       operationId: listUserBillingAccounts
+      parameters:
```

Comment:
Do I need to update orchestration for this too? Or can we just wait and see if anyone ever requests the ability to pass this parameter through orchestration?

https://github.com/broadinstitute/firecloud-orchestration/blob/219bdb1a7c4c85df5350f27be9292bb66233a355/src/main/scala/org/broadinstitute/dsde/firecloud/webservice/UserApiService.scala#L105

Reply:
It might be less overhead to do it now, since there's already a ticket open, and everything is fresh in mind.

Reply:
I'm just not sure if there is any need to expose it through orchestration, and I'd like to avoid the extra work of making changes in another repository unless there is a good reason to do so. The API has been this way for a long time with no complaints, and I don't even know if anyone is actually using the orchestration endpoint.

Reply:
Created https://broadworkbench.atlassian.net/browse/SUP-711 for this.